pkg/rescaffold/migrate.go

```diff
@@ -51,6 +51,10 @@ func (opts *MigrateOptions) Rescaffold() error {
 	if err := kubebuilderInit(config); err != nil {
 		log.Fatal(err)
 	}
+	// call edit subcommands to enable or disable multigroup layout
+	if err := kubebuilderEdit(config); err != nil {
+		log.Fatal(err)
```

Comment:
Nit: Can we also add an error message here so that users are aware when it errored, like:
```
log.Fatalf("Failed to run edit subcommand %v", err)
```

Reply:
Sounds great, it can also applied to the Init command.